code/modules/vending/security.dm

```diff
@@ -49,3 +49,52 @@
 /obj/item/vending_refill/security
 	machine_name = "SecTech"
 	icon_state = "refill_sec"
+
+/obj/machinery/vending/security/proc/redeem_voucher(obj/item/mining_voucher/security/voucher, mob/redeemer)
+	var/obj/item/storage/spawned_box
+
+	var/items = list("Carbon Fibre Sabre", "NPS-10")
+
+	var/selection = tgui_input_list(redeemer, "Pick your equipment", "Voucher Redemption", sort_list(items), "NPS-10")
+	if(!selection || !Adjacent(redeemer) || QDELETED(voucher) || voucher.loc != redeemer)
+		return
+
+	spawned_box = new /obj/item/storage/box/gearvend
```

Comment:
Spawning a box just to insert one item into it is silly for the sword, additionally it also just gives you a box with a non-box sized item in it which allows for you to carry the sword/gun in a backpack.